stories/config.js

```diff
@@ -25,8 +25,12 @@ prepareForScreenshots();
 // automatically import all files ending in *.stories.js
 // $FlowFixMe - require.context seems not correctly typed.
 const req = require.context("../packages", true, /.stories.js$/);
+// $FlowFixMe - require.context seems not correctly typed.
+const reqDocs = require.context("../docs", true, /.stories.js$/);
```

Comment:
both of these should be `\.stories\.js$`

Reply:
Ahh good catch :)